modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/acquisitionannotation/v0.1.0.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpactsh": "{{base}}/schemas/bbp/core/activity/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/acquisitionannotation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id" : "this:AcquisitionAnnotationShape",
+      "@type" : "sh:NodeShape",
+      "label" : "Acquisition and annotation shape definition",
+      "targetClass" : "nsg:AcquisitionAnnotation",
```

Comment:
Shouldn't we have "ImageAcquisitionAnnotation" as targeted type ?

Reply:
Changed the prov pattern to remove "Image"